test/unit/on_config_spec.js

```diff
@@ -117,99 +102,82 @@ describe('UNIT: on_config', function () {
             //     expect($_state.$current.redirectTo).toEqual('/query-builder/mutations');
             // });
 
-            it('should activate redirectTo app.queryBuilder.mutations', function() {
+            it("should activate redirectTo app.queryBuilder.mutations", function() {
                 goTo(_state);
-                expect($_state.current.name).toBe('app.queryBuilder.mutations');
+                expect($_state.current.name).toBe("app.queryBuilder.mutations");
             });
-
         });
 
-
-
-        describe('app.queryBuilder.mutations',()=>{
-            beforeEach(()=>{
+        describe("app.queryBuilder.mutations", () => {
+            beforeEach(() => {
                 _state = "app.queryBuilder.mutations";
             });
 
             it('should respond to URL "/query-builder/mutations"', function() {
                 goTo(_state);
-                expect($_state.href(_state)).toEqual('/query-builder/mutations');
+                expect($_state.href(_state)).toEqual(
+                    "/query-builder/mutations"
+                );
             });
 
-            it('should have a root state', function() {
+            it("should have a root state", function() {
                 goTo(_state);
-                let _parentState = 'app';
+                let _parentState = "app";
                 expect($_state.includes(_parentState)).toBeTruthy();
             });
 
-            it('should have a parent state', function() {
+            it("should have a parent state", function() {
                 goTo(_state);
-                let _parentState = 'app.queryBuilder';
+                let _parentState = "app.queryBuilder";
                 expect($_state.includes(_parentState)).toBeTruthy();
             });
 
-            it('should activate the state', function() {
+            it("should activate the state", function() {
                 goTo(_state);
                 expect($_state.current.name).toBe(_state);
             });
 
-            xit('should have 3 views defined', function() {
+            xit("should have 3 views defined", function() {
```

Comment:
Another one disabled... wonder why these are turned off. If the test is really not needed, then it should be removed. If it's broken and needs to be fixed, it would be nice to have a comment. Maybe there's an issue logged already?